lib/approvals.js

```diff
@@ -9,7 +9,7 @@ const _ = require('lodash')
  *  `mergeable` is false
  */
 module.exports = async (pr, context, settings) => {
-  let approvalSettings = settings.mergeable.approvals
+  let approvalSettings = settings.mergeable.approvals ? settings.mergeable.approvals : 0
```

Comment:
Just a suggestion. But I think this can just be written as `let approvalSettings = settings.mergeable.approvals || 0`

Reply:
you're right